src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -159,6 +162,9 @@ class CoreWorkerDirectTaskSubmitter {
   /// Factory for producing new clients to request leases from remote nodes.
   LeaseClientFactoryFn lease_client_factory_;
 
+  /// Provider of worker leasing decisions.
```

Comment:
Can you clarify this comment to say that it is only used for the decision of which node to request from first?